core/controllers/profile.py

```diff
@@ -233,7 +233,8 @@ class MailingListSubscriptionHandler(
                     'validators': [{
                         'id': 'is_nonempty'
                     }]
-                }
+                },
+                'default_value': None
```

Comment:
I'm a bit confused about this. If the argument is supposed to be a string then is a default_value of None actually allowed? (Is there prior art that shows that this is what should be done in order to treat the argument as optional?)

Reply:
Yep, [here's](https://github.com/oppia/oppia/blob/de8be0bebdd1550dd21e2d2663439c0556ad84c6/core/controllers/payload_validator.py#L94) the line.